### PR TITLE
MGMT-12593: added periodic job for ztp-hypershift-zero-nodes

### PR DESCRIPTION
**Pull request by danielerez**
* Added 'e2e-ai-operator-ztp-hypershift-zero-nodes-periodic' job
  - Running at 2am from Monday through Thursday
* On 'e2e-ai-operator-ztp-hypershift-zero-nodes' job
  - Updated 'run_if_changed'

### CODE REVIEW

**Comment by osherdp** [comment]
@danielerez fix typo "aded" in commit message

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml [hunk 1]
```diff
@@ -245,6 +245,79 @@ periodics:
...
+    repo: assisted-service
+  labels:
+    ci-operator.openshift.io/cloud: packet-edge
+    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci-operator.openshift.io/variant: edge
+    ci.openshift.io/generator: prowgen
+    job-release: "4.12"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-assisted-service-master-edge-e2e-ai-operator-ztp-hypershift-zero-nodes-periodic
+  spec:
```
Comment (osherdp) [inline ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml:264]: sorry about it. for now we have to add this manually:
```suggestion
  reporter_config:
    slack:
      channel: '#assisted-deployment-ci'
      job_states_to_report:
      - failure
      - error
      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
        {{ else }} :red_jenkins_circle: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs>'
  spec:
```

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by osherdp** [comment]
/retitle MGMT-12593: added periodic job for ztp-hypershift-zero-nodes

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml (hunk 1 above)
Comment (osherdp) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml:264]: hopefully soon we'll be able to have a global configuration of this ``reporter_config`` and we won't need to add more (https://github.com/openshift/ci-tools/pull/3140)

Comment (danielerez) [reply ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml:273]: > hopefully soon we'll be able to have a global configuration of this `reporter_config` and we won't need to manually edit this part ([openshift/ci-tools#3140](https://github.com/openshift/ci-tools/pull/3140))

yep, would indeed be nicer:)

**Comment by danielerez** [comment]
@osherdp i've added the reporter config, can u plz take another look?

**Review by osherdp — APPROVED** [review]

**Comment by osherdp** [comment]
/hold
oops, sorry about it. now it will not be needed anymore: https://github.com/openshift/release/pull/34295

**Review by osherdp — APPROVED** [review]
/unhold

**Pull request merged**